x-pack/apm-server/aggregation/spanmetrics/aggregator.go

```diff
@@ -264,13 +271,18 @@ func (a *Aggregator) processDroppedSpanStats(event *model.APMEvent, dss model.Dr
 		dss.DestinationServiceResource,
 		dss.ServiceTargetType,
 		dss.ServiceTargetName,
+
+		// BUG(axw) dropped span statistics do not contain span name.
+		// Capturing the service name requires changes to Elastic APM agents.
+		"",
```

Comment:
My thinking was: the UI needs to decide what to do with older `service_destination` metrics which lack the `span.name` field (using `missing`), so that should also apply in the case where `span.name` is unavailable for aggregation.